packages/freezed/lib/builder.dart

```diff
@@ -10,6 +10,6 @@ Builder freezed(BuilderOptions options) {
   return PartBuilder([FreezedGenerator(options.config)], '.freezed.dart',
       header: '''
 // GENERATED CODE - DO NOT MODIFY BY HAND
-// ignore_for_file: unused_element, deprecated_member_use, deprecated_member_use_from_same_package, use_function_type_syntax_for_parameters, unnecessary_const, avoid_init_to_null, invalid_override_different_default_values_named, prefer_expression_function_bodies
+// ignore_for_file: unused_element, deprecated_member_use, deprecated_member_use_from_same_package, use_function_type_syntax_for_parameters, unnecessary_const, avoid_init_to_null, invalid_override_different_default_values_named, prefer_expression_function_bodies, annotate_overrides
```

Comment:
why is this lint disabled?

Reply:
Because when using mixins, the class overrides the mixxins abstract getters as expected but the linter is unhappy because it wants us to annotate it with `@override`. If I'm not mistaken `@override` is only to give aesthetic and doesn't actually change the behavior of the method right ?

Reply:
I'd still add the necessary `@override` 

I try to avoid disabling lints, so that tests can pick up warnings inside generated code to avoid potential issues.

Reply:
That would need a lot more work for not much. Especially when `@override` is purely optional so I don't know. We'd have to check all `MyClass`'s super classes and compute which ones are being overridden.